packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -253,28 +249,4 @@ export class EVMAAWallet extends LoggerWrapper {
             return this.crossmintService.fetchNFTs(this.account.address, this.chain);
         });
     }
-
-    private getLegacyTransactionFeesParamsIfApply(gasFeeParams?: GasFeeTransactionParams) {
-        const { maxFeePerGas, maxPriorityFeePerGas } = gasFeeParams ?? {};
-
-        if (hasEIP1559Support(this.chain)) {
-            return {
-                // only include if non-null and non-zero
-                ...(maxFeePerGas && { maxFeePerGas: BigInt(maxFeePerGas.toString()) }),
-                ...(maxPriorityFeePerGas && { maxPriorityFeePerGas: BigInt(maxPriorityFeePerGas.toString()) }),
-            };
-        } else {
-            if (maxFeePerGas || maxPriorityFeePerGas) {
-                console.warn(
-                    "maxFeePerGas and maxPriorityFeePerGas are not supported on this chain as it supports Legacy Transacitons. Ignoring them."
-                );
-            }
-            return {
```

Comment:
I could be missing something, but I think EIP1559 may have been conflated w/ the Gelato situation, coincidentally, this has worked fine so far because polygon CDK chains, which ZD uses Gelato for, don't support 1559.